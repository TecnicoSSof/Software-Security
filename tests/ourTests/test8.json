{"ast_type":"Module","body":[{"ast_type":"Assign","col_offset":0,"lineno":1,"targets":[{"ast_type":"Name","col_offset":0,"ctx":{"ast_type":"Store"},"id":"o","lineno":1}],"value":{"args":[],"ast_type":"Call","col_offset":4,"func":{"ast_type":"Name","col_offset":4,"ctx":{"ast_type":"Load"},"id":"get","lineno":1},"keywords":[],"lineno":1}},{"ast_type":"Assign","col_offset":0,"lineno":2,"targets":[{"ast_type":"Name","col_offset":0,"ctx":{"ast_type":"Store"},"id":"x","lineno":2}],"value":{"ast_type":"Name","col_offset":4,"ctx":{"ast_type":"Load"},"id":"o","lineno":2}},{"ast_type":"Expr","col_offset":0,"lineno":3,"value":{"args":[{"ast_type":"Name","col_offset":16,"ctx":{"ast_type":"Load"},"id":"o","lineno":3}],"ast_type":"Call","col_offset":0,"func":{"ast_type":"Name","col_offset":0,"ctx":{"ast_type":"Load"},"id":"send_mail_jinja","lineno":3},"keywords":[],"lineno":3}},{"ast_type":"Expr","col_offset":0,"lineno":4,"value":{"args":[{"ast_type":"Name","col_offset":16,"ctx":{"ast_type":"Load"},"id":"request","lineno":4}],"ast_type":"Call","col_offset":0,"func":{"ast_type":"Name","col_offset":0,"ctx":{"ast_type":"Load"},"id":"send_mail_jinja","lineno":4},"keywords":[],"lineno":4}},{"ast_type":"Expr","col_offset":0,"lineno":5,"value":{"args":[{"ast_type":"Name","col_offset":7,"ctx":{"ast_type":"Load"},"id":"x","lineno":5}],"ast_type":"Call","col_offset":0,"func":{"ast_type":"Name","col_offset":0,"ctx":{"ast_type":"Load"},"id":"RawSQL","lineno":5},"keywords":[],"lineno":5}}]}
