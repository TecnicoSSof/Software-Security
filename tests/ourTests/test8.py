o = get()
x = o
send_mail_jinja(o)
send_mail_jinja(request)
RawSQL(x)