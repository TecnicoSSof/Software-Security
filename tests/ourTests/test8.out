[{"vulnerability":"XSS",
"source":"get",
"sink":"send_mail_jinja",
"sanitizer":""},
{"vulnerability":"XSS",
"source":"request",
"sink":"send_mail_jinja",
"sanitizer":""}]